fgpyo/util/metric.py

```diff
@@ -137,7 +137,7 @@ def values(self) -> Iterator[Any]:
         """An iterator over attribute values in the same order as the header."""
         return iter(attr.astuple(self, recurse=False))
 
-    def formatted_values(self) -> Iterator[str]:
+    def formatted_values(self) -> List[str]:
         """An iterator over formatted attribute values in the same order as the header."""
         return [self.format_value(value) for value in self.values()]
```

Comment:
I think we rely on this API elsewhere, so lets keep it an iterator and instead change the actual return type from a List to a generator (which is an iterator).  I am not sure why mypy didn't find this sooner.

```suggestion
    def formatted_values(self) -> Iterator[str]:
        """An iterator over formatted attribute values in the same order as the header."""
        return (self.format_value(value) for value in self.values())
```